test/cpp/interop/interop_server.cc

```diff
@@ -73,6 +73,18 @@ const char kEchoInitialMetadataKey[] = "x-grpc-test-echo-initial";
 const char kEchoTrailingBinMetadataKey[] = "x-grpc-test-echo-trailing-bin";
 const char kEchoUserAgentKey[] = "x-grpc-test-echo-useragent";
 
+namespace {
+xds::data::orca::v3::OrcaLoadReport TestOrcaReportToXdsOrcaReport(
```

Comment:
Instead of adding this conversion function, I suggest just changing `RecordCallMetrics()` and `RecordServerMetrics()` to take their input as a `TestOrcaReport` proto.

Reply:
Done